h5p-bildetema/src/components/SearchPage/SearchResultCard/SearchResultCard.module.scss

```
@@ -9,6 +10,32 @@
   border: 2px solid $dark-beige;
 }
 
+.bookmarkButton {
+  position: absolute;
+  z-index: $fav-language-selector-z-index;
+  right: 0.25rem;
+  top: 0.25rem;
+
+  & button {
+    all: unset;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    border-radius: 50%;
+    padding: 0.3rem;
```

Make sure the button has a good touch target size
```suggestion
    padding: 0.5rem;
```